src/main/java/com/wfector/command/Help.java

```diff
@@ -2,54 +2,52 @@
 
 import java.util.ArrayList;
 
-import org.bukkit.ChatColor;
+import com.wfector.notifier.ChestShopNotifier;
 import org.bukkit.command.CommandSender;
 
 public class Help {
+    private static ChestShopNotifier plugin;
+
+    public Help(ChestShopNotifier main) {
+        plugin = main;
+    }
 
     public static void SendDialog(CommandSender sender) {
```

Comment:
One last thing that I noticed then it should be good:
You either need to pass the plugin instance to the method or make the method non-static and initialise the Help object in the CommandRunner class in order to access the plugin field. (Help is never initialised in the CommandRunner). I suggest fixing the case of the sendDialog method at the same time.